### PR TITLE
[ADD] osi_analytic_segments_defaults

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
Depends on #117

File: osi_analytic_segments_defaults/README.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Credits
+=======
+
+* Mayank Gosai <mgosai@ursainfosystems.com>
+* Bhavesh Odedra <bodedra@opensourceintegrators.com>
+
+Contributors
+------------
+
+* Open Source Integrators <http://www.opensourceintegrators.com>
```
Comment (mgosai) [inline osi_analytic_segments_defaults/README.rst]: Swap the names between contributors and credits.

**Review by mgosai — CHANGES_REQUESTED** [review]
Swap the names between contributors and credits.

**Comment by murtuzasaleh** [comment]
> Swap the names between contributors and credits.

Done

**Review by mgosai — APPROVED** [review]

**Pull request merged**